### PR TITLE
Lighthouse CI as Page Speed Insight

### PR DESCRIPTION
**Pull request by emersonlaurentino**
## What's the purpose of this pull request?
Use the PSI config in our lhci

## How it works? 
looking at lhci in pull requests from stores

## How to test it?
looking at lhci in pull requests in https://github.com/vtex-sites/marinbrasil.store/pull/270

## References
https://github.com/GoogleChrome/lighthouse
https://github.com/GoogleChrome/lighthouse-ci
https://developers.google.com/speed/docs/insights/v5/about

### CODE REVIEW

File: packages/lighthouse-config/src/index.ts [hunk 1]
```diff
@@ -3,14 +3,39 @@ interface Params {
...
+
 const lhConfig = ({ urls, server }: Params) => {
   const url = urls.map((path) => `${server}${path}`)
 
   return {
     ci: {
       collect: {
         url,
-        numberOfRuns: 5,
+        settings: {
```
Comment (tlgimenes) [inline packages/lighthouse-config/src/index.ts:18]: Aren't these the default config already ? 

Also, we should figure out a way to run two different devices, one using desktop and another using mobile

File: packages/lighthouse-config/src/index.ts [hunk 2]
```diff
@@ -3,14 +3,39 @@ interface Params {
...
+ */
+
 const lhConfig = ({ urls, server }: Params) => {
   const url = urls.map((path) => `${server}${path}`)
 
   return {
     ci: {
       collect: {
         url,
-        numberOfRuns: 5,
```
Comment (tlgimenes) [inline packages/lighthouse-config/src/index.ts:13]: why removing 5 runs ?

**Review by tlgimenes — CHANGES_REQUESTED** [review]

File: packages/lighthouse-config/src/index.ts (hunk 1 above)
Comment (emersonlaurentino) [reply packages/lighthouse-config/src/index.ts:18]: for it, you should use the extends config, like this:

```
// lhci config
  extends: 'lighthouse:default',
``

about run both devices, still is not possible, you can see more about [here](https://github.com/GoogleChrome/lighthouse-ci/issues/138).

File: packages/lighthouse-config/src/index.ts (hunk 2 above)
Comment (emersonlaurentino) [reply packages/lighthouse-config/src/index.ts:13]: because the PSI standard is 3

File: packages/lighthouse-config/src/index.ts (hunk 1 above)
Comment (tlgimenes) [reply packages/lighthouse-config/src/index.ts:18]: Cool ! what about http1 ?

Comment (emersonlaurentino) [reply packages/lighthouse-config/src/index.ts:18]: eita, I had forgotten about it, I'll see what I can do.

**Review by tlgimenes — APPROVED** [review]
It's overall ok for me. However, if you want, you can enhance this PR with performance budgets for both webpack and lighthouse CI

**Comment by emersonlaurentino** [comment]
@tlgimenes yes, I will make this.

File: packages/lighthouse-config/src/index.ts (hunk 1 above)
Comment (emersonlaurentino) [reply packages/lighthouse-config/src/index.ts:18]: done

**Pull request merged**